### ISSUE TITLE
Nothing happen when I click RunCat.exe | 双击 RunCat.exe 后没有任何反应

### ISSUE THREAD

**Issue by SaarChaffee** [Kyome22/RunCat365#73]
My Windows version is 19042.1348, and I have installed .NET 5.0 SDK v5.0.403 which include .NET Runtime 5.0.12.
I have tried to change my .NET Runtime to 5.0.0, and it still nothing running.

**Comment by Aarmii** [comment]
我是windows11，运行RunCat_with_.NET5.0/RunCat.exe没有任何反应

**Comment by SaarChaffee** [comment]
> 

带了.net5.0的也试了我也不行

**Comment by xuxifan** [comment]
编译好的







在 2021-12-09 20:23:30，"SaarChaffee" ***@***.***> 写道：

带了.net5.0的也试了我也不行

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

从网易126邮箱发来的超大附件
Runcat.zip (60.93M, 2022年1月9日 9:13 到期)
下载

**Comment by yunghahn** [comment]
I found a solution.

Go to 1.9 verson, click assets, and download first one.

I am using win11.

Hope it might help.

**Comment by ChenMathilda** [comment]
> I found a solution.
> 
> Go to 1.9 verson, click assets, and download first one.
> 
> I am using win11.
> 
> Hope it might help.

It works for my Win10 PC!!!
Thanks!

**Comment by sfantree** [comment]
Click and Nothing run, or I try to build with compatibility. you can try #84

**Comment by alexya** [comment]
@yunghahn Yes, 1.9 version can work on my win10 PC LTSC 1809 Build 17763.2452

**Comment by Aya-X** [comment]
> I found a solution.
> 
> Go to 1.9 verson, click assets, and download first one.
> 
> I am using win11.
> 
> Hope it might help.


It work for my Win10, many thanks!

**Comment by greenhandatsjtu** [comment]
You can try https://github.com/Kyome22/RunCat_for_windows/releases/tag/2.0 , it now works well on my computer (both 1.10 and 1.11 don't run on my computer).

**Comment by SaarChaffee** [comment]
> Maybe you can try [RunCat-with-dotnet6-x64.zip](https://github.com/Kyome22/RunCat_for_windows/releases/download/2.0/RunCat-with-dotnet6-x64.zip) in latest release 2.0, it now works well on my computer (Windows 11, both 1.10 and 1.11 don't run on it).

Looks like the problem becomes that it can't get my computer name.

**Comment by greenhandatsjtu** [comment]
> > Maybe you can try [RunCat-with-dotnet6-x64.zip](https://github.com/Kyome22/RunCat_for_windows/releases/download/2.0/RunCat-with-dotnet6-x64.zip) in latest release 2.0, it now works well on my computer (Windows 11, both 1.10 and 1.11 don't run on it).
> 
> Looks like the problem becomes that it can't get my computer name.

Hi @SaarChaffee , it seems like an known issue of Windows registry,  you can refer to https://github.com/Kyome22/RunCat_for_windows/issues/19 for details, I'm not sure but this [link](https://stackoverflow.com/questions/17980178/cannot-load-counter-name-data-because-an-invalid-index-exception) may be helpful.

**Comment by SaarChaffee** [comment]
> > > Maybe you can try [RunCat-with-dotnet6-x64.zip](https://github.com/Kyome22/RunCat_for_windows/releases/download/2.0/RunCat-with-dotnet6-x64.zip) in latest release 2.0, it now works well on my computer (Windows 11, both 1.10 and 1.11 don't run on it).
> > 
> > 
> > Looks like the problem becomes that it can't get my computer name.
> 
> Hi @SaarChaffee , it seems like a known issue of Windows registry, you can refer to #19 for details, this [link](https://stackoverflow.com/questions/17980178/cannot-load-counter-name-data-because-an-invalid-index-exception) may be helpful.

Oh I just solved this problem! This is caused by an abnormal shutdown of the computer, such as a sudden power failure on the desktop, so reset the performance counters. And this is my solution to the problem:
- open the PowerShell with Administrator permission
- go to the `C:\Windows\System32` (x86) or `C:\Windows\SysWOW64` (x64)
- write `lodctr  /r` and reboot the computer
- and now it works

And I also saw another solution with one more step:
- `Get-AppxPackage | % { Add-AppxPackage -DisableDevelopmentMode -Register "$($_.InstallLocation)\AppxManifest.xml" -verbose }`
- `lodctr  /r`
- reboot

**Comment by greenhandatsjtu** [comment]
Glad to hear that! Hope you enjoy RunCat :)

**Comment by SaarChaffee** [comment]
> Glad to hear that! Hope you enjoy RunCat :)

thanks a lot

**Comment by SaarChaffee** [comment]
> Glad to hear that! Hope you enjoy RunCat :)

Maybe you can pin this issue to the top, I will translate the title and solution into Chinese.

**Comment by SaarChaffee** [comment]
在 `计算机管理` 里的 `事件查看器`-`Windows日志`-`应用程序` 中如果看到错误日志说 `System.InvalidOperationException: Cannot load Counter Name data because an invalid index '' was read from the registry. Performance counters on the machine may need to be repaired.`，这个是电脑意外断电或别的原因导致的不正常关机引起的性能计数器没办法使用，可以用下面的办法解决：
- 用管理员运行 `PowerShell`
- 进入 `C:\Windows\System32` （32位系统）或者 `C:\Windows\SysWOW64` （64位系统）
- 输入 `lodctr  /r`
- 重启电脑

如果上面的还不能解决，那么重新到刚才的路径中
- 输入`Get-AppxPackage | % { Add-AppxPackage -DisableDevelopmentMode -Register "$($_.InstallLocation)\AppxManifest.xml" -verbose }`
- 然后再 `lodctr  /r`
- 重启电脑

**Comment by SaarChaffee** [comment]
> > > Glad to hear that! Hope you enjoy RunCat :)
> > 
> > 
> > Maybe you can pin this issue to the top, I will translate the title and solution into Chinese.
> 
> Oh sorry, I'm just a contributor, let's ask @Kyome22 for his opinion

刚看到哈哈哈

**Comment by Nicolasking007** [comment]
非常感谢  解决问题
